lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+  def initialize
+    @items = { 'Pepperoni' => 1,
+              'Hot Shot' => 1,
+              'Jardin' => 2,
+              'Provencale' => 2,
+              'Roquefort' => 3,
+              'Neptune' => 3,
+              'Poulet' => 4,
+              'Four Seasons' => 4 }
+  end
+
+  def contains?( name)
+    @items.has_key?(name)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. (https://github.com/bbatsov/ruby-style-guide#hash-key)